Answers.md

```diff
@@ -0,0 +1,15 @@
+Sprint Challenge
+
+1.  Box3 would have the highest specificity as it's last and doubley specific compared to Box.
```

Comment:
Yepp! If all of the elements have the same weight, then the last is regarded as having the most. I'm not too sure about the doubly specific portion but can always get back to you about that.